controllers/admin/AdminSearchController.php

```diff
@@ -239,6 +240,7 @@ public function searchModule()
         $all_modules = Module::getModulesOnDisk(true, Context::getContext()->employee->id);
         foreach ($all_modules as $module) {
             if (stripos($module->name, $this->query) !== false || stripos($module->displayName, $this->query) !== false || stripos($module->description, $this->query) !== false) {
+                // @todo link to module manager with prefilled search
```

Comment:
Please create an issue. Not anymore todo without linked issue.

Reply:
I don't know enough about this feature to add an issue. I just noticed the broken link

Reply:
:thinking: Then what do you think about creating a draft issue ?

Something  like 
> I noticed this linked is broken : 
> ```
> $module->linkto = 'index.php?tab=AdminModules&tab_module=' . $module->tab . '&module_name=' . $module->name . '&anchor=' . ucfirst($module->name) . '&token=' . Tools::getAdminTokenLite('AdminModules');
> ```
> in `AdminSearchController::searchModule`
> I think it should be linked to module manager with prefilled search but it requires more investigation.

Then you add `waiting for dev` and `waiting for PM` tags.

This way we keep a unique central "todo list" with github issues and we assume that we can't always describe perfect expected solution.

What do you think @eternoendless ? ^^

Reply:
The issue existed already https://github.com/PrestaShop/PrestaShop/issues/28303

Reply:
I don't see this PR before, i made one PR for this issue : https://github.com/PrestaShop/PrestaShop/pull/28785